### PR TITLE
Team-based antagonists now require 5+ rounds to be rolled

### PR DESCRIPTION
**Pull request by Zonespace27**
[QoL]
## About the PR 
Requires Nukeops, gang leaders, revheads, and conspirators to have at least 5 rounds played prior to being able to pick them.

## Why's this needed? 
Being a dead weight to your team because you don't know how to shoot a gun is inherently bad design.

## Changelog

```changelog
(u)Zonespace
(*)Players need at least 5 rounds to play team-based antagonist roles.
```

### CODE REVIEW

File: code/datums/preferences.dm [hunk 1]
```diff
@@ -1309,23 +1309,30 @@ datum/preferences
 			HTML += "</td>"
 
 		HTML += "<td valign='top' class='antagprefs'>"
+		if (user?.client?.player.get_rounds_participated() < TEAM_BASED_ROUND_REQUIREMENT && !jobban_isbanned(user, "Syndicate"))
```
Comment (ZeWaka) [inline code/datums/preferences.dm:1312]: What's with the additional jobban check? This is done later.

Comment (Zonespace27) [reply code/datums/preferences.dm:1312]: Done to prevent the double HTML message (round req and ban message)

Comment (ZeWaka) [reply code/datums/preferences.dm:1312]: either have both or save the result of this jobban check so we don't perform it twice

Comment (Zonespace27) [inline code/datums/preferences.dm:1312]:
```suggestion
		if (user?.client?.player.get_rounds_participated() < TEAM_BASED_ROUND_REQUIREMENT)
```

Comment (Zonespace27) [reply code/datums/preferences.dm]: Done

**Pull request merged**